kwalify/component/test_data_validity.py

```diff
@@ -0,0 +1,26 @@
+from glob import iglob
+from pykwalify.core import Core
+import yaml
+
+def get_schema(version):
+    path = 'schemas/kwalify/component/v{}.yaml'.format(version)
+    contents = open(path)
+    return yaml.load(contents)
+
+def create_validator(source_data):
+    version = source_data.get('schema_version', '1.0.0')
+    schema = get_schema(version)
+    validator = Core(source_data={}, schema_data=schema)
+    validator.source = source_data
+    return validator
+
+def test_data_valid():
+    """ Check that the content of data fits with masonry schema v2 """
+    for component_file in iglob('*/component.yaml'):
+        print(component_file)
+        source_data = yaml.load(open(component_file))
```

Comment:
should we have an error in case the file we load isn't valid YAML and it fails to load?

http://stackoverflow.com/questions/1773805/how-can-i-parse-a-yaml-file-in-python